packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -62,6 +34,27 @@ const propTypes = {
     ),
   ),
   /**
+   * An array of nav items to be displayed on the user profile/ settings menu/popup.
+   */
+  profileLinks: PropTypes.arrayOf(PropTypes.shape(
+    navItemShape)),
+  /**
+   * User name to be displayed in the profile in navigation.
+   */
+  userName: PropTypes.string.isRequired,
+  /**
+   * Avatar to be displayed in user profile in navigation.
+   */
+  avatar: PropTypes.PropTypes.element,
+  /**
+   * A unique id set to the profile popup button that will be referred in profile popup.
+   */
+  profileId: PropTypes.string.isRequired,
```

Comment:
we forgot to chat about this in our call, but I had talked with Yao. He just meant profile and help shouldn't declare their own ID's as there is a chance they could be used more than once on a page somehow. but navigatin is a singleton so it would be fine to statically declare the Id's in this root level component rather than the consumer have to pass them down